frontend/src/views/Lobby.tsx

```diff
@@ -114,13 +136,25 @@ function LobbyPage() {
         &quot;.
       </LargeText>
       { error ? <ErrorMessage message={error} /> : null }
+      { loading ? <Loading /> : null }
+
       <div>
         {
           users?.map((user) => (
-            <UserNicknameText onClick={() => deleteUser(user)}>
-              {user.nickname}
-              {user.nickname === host?.nickname ? ' (host)' : ''}
-            </UserNicknameText>
+            <PlayerCard
+              user={user}
+              isHost={user.nickname === host?.nickname}
```

Comment:
This works for now, and honestly should work in the future as well, but I just wanted to note that potentially in the future this could be replaced by an equals-type function.